thevoid/connection.cpp

```diff
@@ -796,6 +869,15 @@ void connection<T>::process_data()
 				send_error(http_response::bad_request);
 				return;
 			} else {
+				CONNECTION_INFO(
+					"received new request: method: %s, url: %s, local: %s, remote: %s, headers: %s",
+					m_access_method.empty() ? "-" : m_access_method.c_str(),
+					m_access_url.empty() ? "-" : m_access_url.c_str(),
+					m_access_local.c_str(),
+					m_access_remote.c_str(),
+					headers_to_string(m_request.headers())
+				);
```

Comment:
You should not use ```.c_str()``` because BH can log every type that has overloaded operator << into std::basic_stream. And std::string has this one.